frontend/src/maps/manage/operations.ts

```diff
@@ -3,9 +3,12 @@ import { nanoid } from "nanoid";
 import { getRoute } from "@/api/map/route_service.ts";
 import type { Point, RouteSegment } from "./structure.ts";
 
-export const useRouteOperations = () => {
-  const [points, setPoints] = useState<Point[]>([]);
-  const [segments, setSegments] = useState<RouteSegment[]>([]);
+export const useRouteOperations = (
```

Comment:
See my comments in RoutePlanningPage on making points[] and segments[] into a new data structure in structure.ts